experimental/traffic-portal/build/traffic_portal_v2.spec

```diff
@@ -25,7 +25,7 @@ License:  Apache License, Version 2.0
 URL:      https://github.com/apache/trafficcontrol/
 Source:   %{_sourcedir}/traffic-portal-%{traffic_control_version}.tgz
 AutoReqProv: no
-Requires: nodejs >= 2:16.14.0
+Requires: nodejs >= 2:18.0.0
```

Comment:
@zrhoffman can we do this? Sorry, I should really just look it up on my own

Reply:
@ocket8888 It's necessary for RHEL 8 support, because the TPv2 integration tests in CDN in a Box fails for NodeJS 16. I completed the TPv2 upgrade to NodeJS 18 in #8005.